packages/front-generator/src/generators/react-typescript/entity-management/template/EntityManagementEditor.tsx.ejs

```diff
@@ -125,7 +125,7 @@ class <%= editComponentClass %>Component extends React.Component<Props & Wrapped
                    <% if (typeof nestedEntityInfo !== 'undefined' && typeof nestedEntityInfo[attr.name] !== 'undefined') {%>
                         nestedEntityView='<%= nestedEntityInfo[attr.name] %>'
                    <% } %>
-                   parentEntityInstanceId={this.props.entityId}
+                   parentEntityInstanceId={this.props.entityId !== <%= className %>.NEW_SUBPATH ? this.props.entityId : undefined}
```

Comment:
`this.props.entityId !== <%= className %>.NEW_SUBPATH`
May be it better to move this in separate method, something like `isNewEntity`?

Reply:
I think in this case it's ok to have explicit code

Reply:
I think it's ok to have an explicit check right here